python/paddle/reader/decorator.py

```diff
@@ -332,6 +335,96 @@ def xreader():
     return xreader
 
 
+def multiprocess_reader(readers, use_pipe=True, queue_size=1000):
+    """
+    multiprocess_reader use python multi process to read data from readers
+    and then use multiprocess.Queue or multiprocess.Pipe to merge all
+    data. The process number is equal to the number of input readers, each
+    process call one reader.
+
+    Use multiprocess.Queue(use_pipe=False, default config) is much more
```

Comment:
The instruction about ```multiprocess.Queue(use_pipe=False, default config) is much faster then multiprocess.``` needs to be updated?

Reply:
done